install-prototool.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+# Installs prototool; used for managing protoc the installation and compilation flags.
+# This script is intended to be run by Travis CI
+
+set -ex
+
+curl -sSL \
+  https://github.com/uber/prototool/releases/download/v1.7.0/prototool-$(uname -s)-$(uname -m) \
```

Comment:
Do we want to parametrise the version as well?

Reply:
Aye, probably should host it into the travis.yaml - also my build is failing, presumably because I can't write to `/usr/local/bin` on a travis host.